src/Components/Onboarding/Steps/Artists/ArtistSearchResults.tsx

```diff
@@ -25,14 +25,15 @@ class ArtistSearchResultsContent extends React.Component<RelayProps, null> {
   }
 
   onArtistFollowed(artistId: string, store: RecordSourceSelectorProxy, data: SelectorData): void {
-    const suggestedArtist = store.get(data.followArtist.artist.related.suggested.edges[0].node.__id)
-    this.excludedArtistIds.add(suggestedArtist.getValue("_id"))
+    const suggestedArtistNode = data.followArtist.artist.related.suggested.edges[0]
```

Comment:
this is really `suggestedArtistEdge` #minor

Reply:
ah right. I'll rename next time I touch this file.